token/js/test/e2e-2022/tokenMetadata.test.ts

```diff
@@ -51,7 +51,7 @@ describe('tokenMetadata', async () => {
     beforeEach(async () => {
         mint = Keypair.generate();
 
-        const mintLen = getMintLen(EXTENSIONS);
+        const mintLen = getMintLen(EXTENSIONS) + 1;
```

Comment:
Looks like this snuck in, can you revert it?

Reply:
You were too quick, lol. It's out.